app/views/ChangeAvatarView/AvatarSuggestion.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useState, useEffect } from 'react';
+import { Text, View } from 'react-native';
+
+import { IAvatar, IUser } from '../../definitions';
+import { Services } from '../../lib/services';
+import I18n from '../../i18n';
+import Avatar from '../../containers/Avatar';
+import styles from './styles';
+import { useTheme } from '../../theme';
+import Touch from '../../containers/Touch';
+
+const Item = ({ item, onPress, text, testID }: { item?: IAvatar; testID?: string; onPress: Function; text?: string }) => {
+	const { colors } = useTheme();
+
+	return (
+		<Touch
+			key={item?.service}
+			testID={testID}
+			onPress={() => onPress(item)}
+			style={[styles.avatarButton, { backgroundColor: colors.borderColor }]}
```

Comment:
`avatarButton` for an `Item` component feels not so good.
If you create a single component per file and have one StyleSheet for each file, you'll it's going to be easier to create meaningful style names.